CMakeLists.txt

```diff
@@ -0,0 +1,117 @@
+cmake_minimum_required(VERSION 2.8.12)
+
+project(cctz)
+
+set(CMAKE_MODULE_PATH
+  ${PROJECT_SOURCE_DIR}/cmake
+  ${PROJECT_SOURCE_DIR}/cmake/modules
+  ${CMAKE_MODULE_PATH})
+
+include(CheckCXXCompilerFlag)
+include(CTest)
+include(GNUInstallDirs)
+
+option(BUILD_EXAMPLES "Whether or not to build examples" ON)
+
+if (CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang")
+  add_compile_options(-Wall)
+endif()
+
+function(cctz_target_set_cxx_standard target)
+  set(cctz_cxx_standard 11)
```

Comment:
I will ignore, not because I feel strong about it, but because I have a reasoning which you might consider okay once I share it with you.

I like to name my project/directory variables uppercase, because the scope is somewhat global, inherited from directory to directory.
Here the variable is in the function local scope, very localized, so I feel like the name does not need to be so distinctive.

Checking a few CMake modules, not too dated, such as FindThreads.cmake, FindGTest.cmake, FindProtobuf.cmake, FindPythonLibs.cmake, I would say it is somewhat similar.
Eventually they also add an underscore prefix.
I guess I could use something like `_cxx_standard` if I wanted to copy the style.

Reply:
Sounds reasonable.